src/main/webapp/meal-page-script.js

```diff
@@ -25,6 +36,8 @@ function fetchMealInfo() {
             ingredientsElement.appendChild(createElementByTag(encodingCheck(ingredient), 'li'));
         }
         createMap();
+    }).catch((error) => {
+        console.log(error);
```

Comment:
Add a TODO for the Alpha product: if the endpoint returned "not found", display an error on the page explaining what happenned. For other types of errors, display a generic "something went wrong" error page.